pkg/utils/utils.go

```diff
@@ -172,3 +173,44 @@ func GetNumVpnConnections(cloud1, cloud2 string) int {
 	}
 	return 1
 }
+
+// DoCIDROverlap returns false if cidr blocks don't share a single ip,
+// i.e. they don't overlap.
+func DoCIDROverlap(cidr1, cidr2 string) (bool, error) {
+	netCIDR1, err := netip.ParsePrefix(cidr1)
+	if err != nil {
+		return true, err
+	}
+	netCIDR2, err := netip.ParsePrefix(cidr2)
+	if err != nil {
+		return true, err
+	}
```

Comment:
Why does it make sense to return true if we fail to parse either IP?

Reply:
I don't think there's a particular reason/advantage to return either true or false in this case because the idea is to check for no errors before using result value. However, I'd understand true in the sense that this function might be used in the context of ensuring that some ip ranges don't overlap (so true, signifying an overlap might be good?)